package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "xata",
-  "displayName": "xata-vscode-extension",
+  "displayName": "Xata",
   "repository": "https://github.com/xataio/vscode-extension",
-  "icon": "doc/xata-icon-128.png",
+  "icon": "https://github.com/xataio/vscode-extension/raw/main/doc/xata-icon-128.png",
```

Comment:
This can change if `main` changes. Use the permalink instead please (press <kbd>y</kbd> on the GitHub page of the file)

Reply:
This need to be relative actually, vsce refuse to package an absolute link here 😅

Reply:
![Travis Kelce exploding head](https://i.giphy.com/media/csSic5XbmNqK8ZUNZN/giphy-downsized.gif)

Reply:
> This need to be relative actually, vsce refuse to package an absolute link here 😅

my VSCode is squawking saying it needs an `https` url (literally that)

Reply:
for the record:
@fabien0102 was right, VS Code was wrong.